projects/cdk/tokens/active-element.ts

```diff
@@ -44,16 +45,36 @@ export const TUI_ACTIVE_ELEMENT = new InjectionToken<Observable<EventTarget | nu
                     map(() => document.activeElement),
                     filter(element => !!element && element.matches('iframe')),
                 ),
-                focusin$.pipe(map(({target}) => target)),
+                focusin$.pipe(
+                    switchMap(event => {
+                        const target = getActualTarget(event);
```

Comment:
you do not need to get actual target here if `root` does not equal `document`

Reply:
But I need it to get `root` :)